staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1399,7 +1399,7 @@ message Image {
     repeated string repo_tags = 2;
     // Digests by which this image is known.
     repeated string repo_digests = 3;
-    // Size of the image in bytes. Must be > 0.
+    // Size of the image in bytes. Must be > 0 and refer to the uncompressed size (i.e. on disk)
```

Comment:
wait, does the `crictl` shows compressed size?

Reply:
https://github.com/kubernetes-sigs/cri-tools/issues/1264

Reply:
Ok, I'm apparently dumb and essentially posted an opposite fix to what I should have done. Although now I have a doubt since it turns out CRI-O does indeed use the uncompressed size, it's only containerd who doesn't:

containerd/containerd#9261

Should we still proceed with this?

Reply:
Apparently now only containerd uses the compressed size; it turns out CRI-O does use the uncompressed size, yielding the same result as Docker

containerd/containerd#9261

At this point I think we should just default it to show the uncompressed, fix it in containerd and then do a KEP as we discussed